pkg/tagger/local/tagstore.go

```diff
@@ -92,6 +91,10 @@ func (s *tagStore) processTagInfo(tagInfos []*collectors.TagInfo) {
 			log.Tracef("processTagInfo err: empty source name, skipping message")
 			continue
 		}
+		if info.SkipCache {
```

Comment:
would it be useful to add some telemetry here?

Reply:
Added!